.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+package-lock.json
+hint-report/
```

Comment:
- [ ] Please 🙏🏽 just the node modules folder must be in the gitignore file. Take out the other files.